packages/react-server/README.md

```diff
@@ -1,169 +1,33 @@
-		// Start the server and listen on port.
-		let httpServer = http.createServer(server).listen(port, function () {
-			console.log("Started!");
-		});
-
-		console.log('Listening on port ' + port);
-
-8. Start the server:
-
-		node server.js
-
-9. Load the page in your browser, either by opening your favorite browser and
-navigating to http://localhost:3000, or by opening it in your default browser from the command line:
-
-		open http://localhost:3000
-
-## Contributing
-We welcome contributions to react-server!  To contribute, follow these steps:
-
-1. Check out the code.  You'll want to [fork the repository](https://help.github.com/articles/fork-a-repo/),
- and then clone it locally.
-
-2. Make your changes!
-
-3. Write some tests. See [the testing guide](/core/test/README.md).
-
-4. Make sure _all_ the tests pass!
-		gulp test
-
-5. Make sure you've followed our style guide.  The style guide is codified as eslint
- rules in [package.json](package.json).  You can read more about what those rules
- do on the [eslint rules page](http://eslint.org/docs/rules/), or with a combination
- of matching the style of the rest of the project and trial and error.
-		gulp eslint
-
-6. Create a [pull request](https://help.github.com/articles/using-pull-requests/)
-
-7. Repeat zero or more times: Get some feedback from [another contributor](https://github.com/redfin/react-server/graphs/contributors),
- resolve it, update your pull request.
-
-8. Your pull request gets merged!  Congratulations, you're officially a react-server contributor.
- Have a 🍺to celebrate; your check is in the mail, we swear 😉.
-
-// TODO: Concepts
-// TODO: routing guide
-// TODO: generator
-
-## What is "triton"? Why do I see that in the git history?
+`react-server` is an [Express](http://expressjs.com/) [middleware](http://expressjs.com/guide/using-middleware.html)
+for serving universal (isomorphic) JavaScript applications built with [React](https://facebook.github.io/react/)
+based on the [Flux pattern](https://facebook.github.io/flux/docs/overview.html).
+Read more on [our dev blog](https://www.redfin.com/devblog/2015/09/thoughts-on-transitioning-to-universal-javascript.html).
 
-Back when we started this project, we came up with a great name -- `triton` -- and we started using it internally. And then it took us forever to get around to making the source code public. And then [Joyent](https://www.joyent.com) released a product called `triton` and stole our thunder, so we had to go with `react-server` instead (a terrible, terrible tragedy, we know).
+## Getting started
 
-[![Dependency Status](https://david-dm.org/redfin/react-server.svg)](https://david-dm.org/redfin/react-server)
-[![devDependency Status](https://david-dm.org/redfin/react-server/dev-status.svg)](https://david-dm.org/redfin/react-server#info=devDependencies)
+For now it's easiest to get started with
+[`react-server-cli`](https://github.com/redfin/react-server/tree/master/packages/react-server-cli).
```

Comment:
I would link to npmjs.org/package/react-server-cli ; the easiest way to get started is not to learn about how monorepos work.